tsconfig.json

```diff
@@ -11,8 +11,5 @@
     "test/**/*.ts",
     "system-test/*.ts",
     "system-test/**/*.ts"
-  ],
-  "exclude": [
-    "node_modules"
```

Comment:
we've been a little willy-nilly about requiring ES2018, ultimately it would seem this isn't required?